packages/core/src/common/severity.ts

```diff
@@ -30,28 +31,28 @@ export enum Severity {
 }
 
 export namespace Severity {
-    const error = 'Errors';
-    const warning = 'Warnings';
-    const info = 'Info';
-    const log = 'Log';
-    const ignore = 'All';
+    const error = nls.localize('theia/core/severity/errors', 'Errors');
+    const warning = nls.localize('theia/core/severity/warnings', 'Warnings');
+    const info = nls.localize('theia/core/severity/info', 'Info');
+    const log = nls.localize('theia/core/severity/log', 'Log');
+    const ignore = nls.localize('theia/core/severity/all', 'All');
```

Comment:
Suggestion: I'm not too sure whether we should localize these values directly. I would rather have a `Severity.toLocaleString()` function that accepts a `string | Severity` and returns a localized version of the string. Use the result at the caller site of `debug-console-contribution.tsx`.